tests/test_web_log.py

```diff
@@ -43,13 +43,31 @@ def test_access_logger_format() -> None:
     Ref: https://stackoverflow.com/a/46102240/595220
     """,  # noqa: E501
 )
-def test_access_logger_atoms(mocker) -> None:
-    utcnow = datetime.datetime(1843, 1, 1, 0, 30)
+def test_access_logger_time(mocker) -> None:
+    now = datetime.datetime(1843, 1, 1, 0, 30)
     mock_datetime = mocker.patch("aiohttp.datetime.datetime")
+    mock_timezone = mocker.patch("aiohttp.timezone")
+    mock_datetime.now.return_value = now
+    mock_timezone.return_value = 28800
+    log_format = '%t'
+    mock_logger = mock.Mock()
+    access_logger = AccessLogger(mock_logger, log_format)
+    request = mock.Mock()
+    response = mock.Mock()
+    access_logger.log(request, response, 3.1415926)
+    assert not mock_logger.exception.called
+    expected = ('[01/Jan/1843:00:29:56 +0000]')
```

Comment:
no need for braces here
```suggestion
    expected = '[01/Jan/1843:00:29:56 +0000]'
```